.github/workflows/build-container.yml

```diff
@@ -0,0 +1,33 @@
+name: dispatched build-docker-image
+
+on:
+  repository_dispatch:
+    types:
+      - dispatch-build
+
+jobs:
+  dispatched-build:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: checkout frontier-squid
+      uses: actions/checkout@v2
+
+#     with:
+#       ref: ${{ github.event.client_payload.ref }}
+#
+#   - run: echo ${{ github.event.client_payload.sha }}
```

Comment:
Yeah, go ahead and drop these comments. We don't provide the ref in the payload from https://github.com/opensciencegrid/docker-software-base/pull/9/ and it's not relevant for cloning this repo anyway.

Reply:
Dropped.